src/image/image.js

```diff
@@ -175,14 +175,25 @@ p5.prototype.saveCanvas = function() {
 
   // copy arguments to array
   const args = [].slice.call(arguments);
-  let htmlCanvas, filename, extension;
+  let htmlCanvas, filename, extension, temporaryGraphics;
 
   if (arguments[0] instanceof HTMLCanvasElement) {
     htmlCanvas = arguments[0];
     args.shift();
   } else if (arguments[0] instanceof p5.Element) {
     htmlCanvas = arguments[0].elt;
     args.shift();
+  } else if (arguments[0] instanceof p5.Framebuffer) {
+    const framebuffer = arguments[0];
+    temporaryGraphics = createGraphics(framebuffer.width, framebuffer.height);
```

Comment:
Ok I think this all works in global mode! The last thing to do is to also make it work in instance mode, where `createGraphics` isn't globally defined. I think in that case using `this.createGraphics` will work though? Maybe try that and try using this in an instance mode sketch to verify